src/containerapp/azext_containerapp/_params.py

```diff
@@ -440,3 +440,12 @@ def load_arguments(self, _):
         c.argument('logger_name', help="The logger name.")
         c.argument('logger_level', arg_type=get_enum_type(["off", "error", "info", "debug", "trace", "warn"]), help="Set the log level for the specific logger name.")
         c.argument('all', help="The flag to indicate all logger settings.", action="store_true")
+
+    with self.argument_context('containerapp debug') as c:
+        c.argument('container', help="The container name that the ephemeral container will target to.")
+        c.argument('replica',
+                   help="The name of the replica. List replicas with 'az containerapp replica list'. A replica may not exist if there is no traffic to your app.")
```

Comment:
```suggestion
                   help="The name of the replica. List replicas with 'az containerapp replica list'. A replica may be not found when it's scaled to zero if there is no traffic to your app.")
```

Reply:
Do we have default option for replica? We should add `Defaults to xxx` if we do.